src/pages/program/[slug]/chat-requests.tsx

```diff
@@ -1,7 +1,7 @@
 import React from "react";
 import { Text } from "../../../components/atomic";
 import { ChatRequestFilterer } from "../../../components/ReviewChats/ChatRequestFilterer";
-import { useCurrentProfile } from "../../../hooks";
+import { AuthorizationLevel, useCurrentProfile } from "../../../hooks";
```

Comment:
Isn't this semantically strange since hooks usually take the form of `useSomething`?

Reply:
Yes. I think in the future, we need to figure out how to consolidate all of our defined types in a more organized manner.